files/metainfo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.clashcross.clashcross</id>
+
+  <name>ClashCross</name>
+  <summary>ClashCross is a cross-platform proxy debugging application built on the Clash core, supporting Linux.To run on Flathub, you must manually set up the system proxy. This is determined by the Flathub mechanism.</summary>
```

Comment:
this is not a proper summary.

Also "cross platform" when it is limited to x86_64 is misleading.

Reply:
The summary is too long as well.

Reply:
hasn't been solved.